test/test/lock/LockTests.java

```diff
@@ -48,4 +19,16 @@ public void datagramSocketLock(TestProcess p) throws Exception {
         out = p.profile("-e lock -d 3 -o collapsed");
         assert out.samples("sun/nio/ch/DatagramChannelImpl.send") > 10;
     }
+
+    @Test(mainClass = RaceToLock.class, inputs = { "0" }, args = "1000", output = true)
+    @Test(mainClass = RaceToLock.class, inputs = { "10000" }, args = "1000", output = true)
+    @Test(mainClass = RaceToLock.class, inputs = { "1000000000" }, args = "1000", output = true)
```

Comment:
Do we need to pass arguments, if they are always the same?

Reply:
Removing the arg, we can use 1s for all cases.